integrator/project_original/src/Pyz/Zed/TestIntegratorWirePlugin/TestParentIntegratorWirePluginDependencyProvider.php

```diff
@@ -9,6 +9,11 @@
 
 class TestParentIntegratorWirePluginDependencyProvider
 {
+    public function getSinglePluginInterface(): SingleWirePluginInterface
```

Comment:
Please add full name to the return type here 
example `\Pyz\Zed\TestIntegratorWirePlugin\SingleWirePluginInterface`